checkstyle.xml

```diff
@@ -47,12 +51,6 @@
       <property name="processJavadoc" value="true"/>
     </module>
 
-
-    <!-- Checks for Size Violations.                    -->
-    <!-- See http://checkstyle.sf.net/config_sizes.html -->
-    <module name="LineLength">
-      <property name="max" value="140"/>
-    </module>
```

Comment:
checkstyle is bundled with Gradle. In newer versions, `cacheFile` and `LineLength` have been moved one level above.